modules/nextflow/src/main/groovy/nextflow/cloud/aws/batch/S3Helper.groovy

```diff
@@ -43,6 +43,40 @@ class S3Helper {
             unset IFS
         }
         
+        nxf_retry() {
+            local max_attempts=\${ATTEMPTS-4}
+            local timeout=\${TIMEOUT-1}
+            local attempt=0
+            local exitCode=0
+            echo "Downloading" \$2
+
+            while [[ \$attempt < \$max_attempts ]]
+            do
```

Comment:
Number comparison in Bash syntax is `(( x < y ))`. See https://stackoverflow.com/a/18668580/395921

Reply:
Fixed